src/content/docs/misc/downloads-api.mdx

```diff
@@ -80,3 +94,116 @@ fi
 This is the most common use case for the API. It will download the latest stable build for the given project and
 Minecraft version. You should always serve & use the stable builds. Experimental builds are prone to error and
 do not receive support.
+
+---
+
+Fill also supports a GraphQL API, which can be accessed at `https://fill.papermc.io/graphql`.
+Common API tools such as Postman will introspect the API and provide a UI for building queries.
```

Comment:
could mention `https://fill.papermc.io/graphiql?path=/graphql`

Reply:
TIL we have a playground url